infra/README.md

```diff
@@ -1,5 +1,23 @@
 # symptomradar infra
 
+## Getting started
+
+Follow these instructions to set up your environment so that you can contribute to the development of the project.
+
+1. Install Terraform. On macOS, the simplest way is probably `brew install terraform`.
```

Comment:
I tried to look for _Getting started_ page on Terraform, but sadly there isn't a good one. Closest thing is _Downloads_. But despite that, maybe add a link to Terraform main page https://www.terraform.io/ (unless this was already in `/README.md` but duplication doesn't hurt in this case)

Reply:
Good point, will do.